qrexec-lib/gentbl.py

```diff
@@ -19,7 +19,7 @@ def print_interval(interval, last_cat):
         else:
             # Allow the interval consisting only of 0x20, to allow spaces in
             # file names
-            if (last_cat[0] in ('S', 'C', 'Z')) and interval[1] > 0x7E:
+            if (last_cat == 'Mc' or last_cat[0] in ('S', 'C', 'Z')) and interval[1] > 0x7E:
```

Comment:
What about other combining characters (other categories "M") ? Looks like safer to exclude all of them for now.

Reply:
This turns out to be a no-op (the generated allowlist is unchanged), presumably because other characters are blocked elsewhere.  That said, I agree that this is more robust.